fleet/comms.go

```diff
@@ -69,6 +71,68 @@ type fleetCommsService struct {
 	agentPubSub mfnats.PubSub
 }
 
+func (svc fleetCommsService) NotifyGroupDatasetEdit(ctx context.Context, ag AgentGroup, datasetID string, policyID string, ownerID string) error {
+	p, err := svc.policyClient.RetrievePolicy(ctx, &pb.PolicyByIDReq{PolicyID: policyID, OwnerID: ownerID})
+	if err != nil {
+		return err
+	}
+
+	var pdata interface{}
+	if err := json.Unmarshal(p.Data, &pdata); err != nil {
+		return err
+	}
+
+	dataset, err := svc.policyClient.RetrieveDataset(ctx, &pb.DatasetByIDReq{
```

Comment:
we shouldn't need to call this, it should get all information it needs from the redis stream